rfc/README.md

```diff
@@ -0,0 +1,42 @@
+# Request for Comments (RFC)
+
+This directory hosts proposals (RFCs) of evolutionary topics around VAST. The
+objective is to actively include the community in the discussion.
+
+## Provide comments to an open RFC
+
+Every RFC has a corresponding pull request with an `rfp` label.
+
+Here are the direct links to open, closed, or all RFCs:
+
+- [Open RFCs][open-rfcs]
```

Comment:
Why not using the PR status to distinguish between open and closed?

Reply:
These are just short links to open/closed PRs, so I think this is actually implemented in the way you are suggesting it.

Reply:
Indeed, sorry for the noise I got confused. Feel free to resolve the comment (I don't seem to have permission to do that)

Reply:
> I don't seem to have permission to do that

Huh, even though it's yours? @dominiklohmann, our GitHub expert...is that normal or are our restrictions tighter than the defaults?

Reply:
It's the default, only people that are authorized for reviews can. That requires at least Triage level permissions on the repository.